### PR TITLE
Simple buttons - grayline updates

### PR DESCRIPTION
**Pull request by chri2547**
image

### CODE REVIEW

File: _component_design/buttons-simple.md [hunk 1]
```diff
@@ -61,6 +61,26 @@ Consider the following best practices when you construct a button:
...
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
 ## Button labels
```
Comment (palindromebart) [inline _component_design/buttons-simple.md]: button labels should go below the specifications, shouldn't it?

Comment (chri2547) [reply _component_design/buttons-simple.md]: @palindromebart, this is all set now. Please LGTM! Woot!

**Comment by palindromebart** [comment]
spacing large button grayline label line is too long

**Comment by chri2547** [comment]
@palindromebart, the style visual has been updated. We're ready for content-LGTM.

**Comment by palindromebart** [comment]
LGTM (with comments)

File: _component_design/buttons-simple.md [hunk 2]
```diff
@@ -108,6 +98,41 @@ Use the following specifications when you construct a simple button.
...
+| Create | Creates a new object. |
+| Delete | Deletes an existing object. The label can include the object being deleted, such as **Delete Image**. |
+| Download | Transfers a file to a user's local system. |
+| Edit | Allows the user to modify data or values. Edit does not imply that the object is automatically updated. In most cases, you still must update in order for the change to occur. |
+| Filter | Changes view to include only objects that match the filter criteria. |
+| Import | Includes objects from an external source. |
+| Manage | Modifies existing object settings or options. |
+| Save | Saves data entered when the user creates an object. |
+| Update | Automatically changes the data of an existing object. |
+{: class="hxTable"}
```
Comment (nimbinatus) [inline _component_design/buttons-simple.md]: s/`{: class="hxTable"}`/`{: .hxTable .docTableFluid }` (part 2 of spacing fix)

File: _component_design/buttons-simple.md [hunk 3]
```diff
@@ -108,6 +98,41 @@ Use the following specifications when you construct a simple button.
...
+### Button labels
+
+Use consistent labels across an action for a UI. Refer to this table when you write button labels.
+
+{% endcolumn %}
+
+{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+{% figure [caption:"Button action labels"] [class:"image bg-light border"] %}
+| Action label | Typical usage  |
```
Comment (nimbinatus) [inline _component_design/buttons-simple.md]: s/`Action label`/`Action&nbsp;label` (part 1 of spacing fix)

**Pull request merged**